### PR TITLE
Updated GIE docs

### PR DESCRIPTION
**Pull request by thobalose**
Updated GIE build docs.

### CODE REVIEW

**Review by hexylena — COMMENTED** [review]
Looks great!

**Review by hexylena — APPROVED** [review]

File: doc/source/admin/special_topics/interactive_environments.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ How GIEs Work
 -------------
 
 A GIE is primarily composed of a Docker container, and the Galaxy visualization
-component. Galaxy vizualisation plugins are rendered using Mako templates and
+component. Galaxy visualisation plugins are rendered using Mako templates and
```
Comment (dannon) [inline doc/source/admin/special_topics/interactive_environments.rst]: s/visualisation/visualization   (I know, it's the en-US spelling, but that's what we've standardized on)

**Comment by dannon** [comment]
@thobalose Thanks for updating all this -- the code example being out of date is my fault, I should have updated it when I worked on that interface, so thank you for catching it!

**Comment by thobalose** [comment]
@dannon @erasche Thank you for all the work. 

I had also made a few changes on https://github.com/galaxyproject/galaxy/pull/6840

**Comment by martenson** [comment]
Thanks @thobalose !

**Pull request merged**